gcloud-java-contrib/gcloud-java-nio/src/test/java/com/google/gcloud/storage/contrib/nio/CloudStorageFileAttributesTest.java

```diff
@@ -29,14 +31,13 @@
 
   private static final byte[] HAPPY = "(✿◕ ‿◕ )ノ".getBytes(UTF_8);
 
-  @Rule
-  public final AppEngineRule appEngineRule = new AppEngineRule();
 
   private Path path;
   private Path dir;
 
   @Before
   public void before() throws Exception {
+    CloudStorageFileSystemProvider.setGCloudOptions(LocalGcsHelper.options());
```

Comment:
Though less important in tests Tests are still java code.

When you explicitly declare a method to throw a checked exception the reader/maintainer
assumes that some code in that method can throw that exception (it is valid use-case
for the test to fail if that happen). 

I think that this is a bad practice (which is in par in my mind to redundant imports) and I don't
really see why tests should be an exception for that.